gr-utils/modtool/CMakeLists.txt

```diff
@@ -28,6 +28,10 @@ install(
     DESTINATION ${GR_PREFSDIR}
 )
 
+install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/support/_gr_modtool
+  DESTINATION ${GR_DATA_DIR}/zsh/site-functions
```

Comment:
Before I do something stupid:

@maitbot is /usr/share/zsh/site-functions for ZSH completion scripts in line with debian packaging rules?